django_spanner/schema.py

```diff
@@ -117,14 +117,24 @@ def create_model(self, model):
             # Create a unique constraint separately because Spanner doesn't
             # allow them inline on a column.
             if field.unique and not field.primary_key:
-                self.deferred_sql.append(
-                    self._create_unique_sql(model, [field.column])
-                )
+                if USING_DJANGO_4:
+                    self.deferred_sql.append(
+                        self._create_unique_sql(model, [field])
+                    )
+                else:
+                    self.deferred_sql.append(
+                        self._create_unique_sql(model, [field.column])
+                    )
```

Comment:
feel it should be:
```
if USING_DJANGO_3:
  xxx
else:
  yyy
```
as future django version will be more likely compatible with 4

Reply:
Thanks for pointing out. Made changes throughout